engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/RuntimeManagementTest.scala

```diff
@@ -81,7 +81,11 @@ class RuntimeManagementTest extends InterpreterTest {
       totalOut = consumeOut
       while (totalOut.length < expect && round < 500) {
         round = round + 1
-        if (round % 10 == 0) forceGC();
+        if (round % 10 == 0) {
+          forceGC();
+          val res = eval("main a b = a * b").execute(7, 6)
+          assertResult(42)(res.asInt)
```

Comment:
However, to execute the _"finalizer"_ some Truffle code must be executing. It wasn't in case of `RuntimeManagementTest`. To mitigate that, I am submitting some simple computation here. As soon as the computation reaches [TruffleSafepoint](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/TruffleSafepoint.html), the _"finalizers"_ are executed and the test can finish.